libs/coin-evm/docs/prepareTransaction.md

```diff
@@ -0,0 +1,44 @@
+# `prepareTransaction.ts`
```

Comment:
there is already doc in the code let's merge infos

Reply:
Same answer as above